addons/web/static/src/views/form/form_controller.js

```diff
@@ -236,6 +240,16 @@ export class FormController extends Component {
             this.buttonBoxTemplate = buttonBoxTemplates.ButtonBox;
         }
 
+        const xmlDocHeader = this.archInfo.xmlDoc.querySelector("header");
+        if (xmlDocHeader) {
```

Comment:
only do that in mobile?

Reply:
But it's in the FormController's `setup()`, so it won't be properly applied in case of resize.

Reply:
This may be lazy computed on resize I think
```js
setup() {
    this.computeMobileHeaderButton();
    onResize(() => this.computeMobileHeaderButton());
}

computeMobileHeaderButton() {
     if (isSmall && !this.statusBarDropdownItemsTemplate) {
          // compute here
     }
}
```

What do you think?

Reply:
I still have the feeling that it adds complexity for not a lot of gain...
but, well, tell me if you prefer that I change it.

Reply:
ok